### ISSUE TITLE
'int' object has no attribute 'strip' (line 792)

### ISSUE THREAD

**Issue by AveCeasar** [XavierXiao/Dreambooth-Stable-Diffusion#32]
Traceback (most recent call last):
  File "main.py", line 792, in <module>
    ngpu = len(lightning_config.trainer.gpus.strip(",").split(','))
AttributeError: 'int' object has no attribute 'strip'

I'm using this fork: https://github.com/gammagec/Dreambooth-SD-optimized

**Comment by SodyFX** [comment]
Make sure in the command that --gpus 0,  has the comma after the zero.

**Comment by AveCeasar** [comment]
yeah, I've already tried it (saw a closed issue here). The full command looks like this:

python main.py --base D:\\Dreambooth-SD-optimized\\configs\\stable-diffusion\\v1-finetune_unfrozen.yaml -t --actual_resume D:\\stable-diffusion-automatic\\model.ckpt -n project01 --gpus 0, --data_root D:\\Dreambooth-SD-optimized\\training --reg_data_root D:\\Dreambooth-SD-optimized\\regimg --class_word girl

**Comment by AcelisWeaven** [comment]
Hi @AveCeasar, I had the same issue in Windows PowerShell and the solution was to replace `--gpus 0,` by `--gpus "0,"`.

**Comment by RowanUnderwood** [comment]
> Hi @AveCeasar, I had the same issue in Windows PowerShell and the solution was to replace `--gpus 0,` by `--gpus "0,"`.

OMG I knew it was something stupid like this.  Yup, this solved my issue also.  Thanks!

**Comment by TemporalLabsLLC-SOL** [comment]
WOW

nice.

**Comment by TemporalLabsLLC-SOL** [comment]
> Hi @AveCeasar, I had the same issue in Windows PowerShell and the solution was to replace `--gpus 0,` by `--gpus "0,"`.

Where did you find this?

**Comment by AcelisWeaven** [comment]
Trial and error haha.

**Comment by Ninja-Ferret** [comment]
@AcelisWeaven Thank you so much!!! (--gpu "0,") fixed it for me!
